README.md

```diff
@@ -501,6 +501,55 @@ The IP configuration supports the following options:
     Note that Classless inter-domain routing (CIDR) notation or network mask notation
     are not supported yet.
 
+- `routing_rule`
+
+    The policy routing rules can be specified via a list of rules given in the
+    `routing_rule` option, which allow routing the packets on other packet fields
+    except for destination address. The default value is a an empty list. Each rule is
+    a dictionary with the following entries:
+   - `priority` -
+      The priority of the rule. A valid priority ranges from 0 to 4294967295. Higher
+      number means lower priority.
+   - `action` -
+      The action of the rule. The possible values are `to-table` (default),
+      `blackhole`, `prohibit`, `unreachable`.
+   - `dport`-
+      The range of the destination port. The range consists of start and end, which can
+      be specified in the form of `$start - $end`. The valid range value has to falls
+      within the range `0 - 65534`.
+   - `family` -
+      The IP family of the rule. The possible values are `ipv4`, `ipv6`.
+   - `from` -
+      The source address of the packet to match (e.g. `192.168.100.58/24`).
+   - `fwmark` -
+      The fwmark value of the packet to match.
+   - `fwmask` -
+      The fwmask value of the packet to match.
+   - `iif` -
+      Select the incoming interface name to match.
+   - `invert` -
+      Invert the selected match of the rule.
```

Comment:
Please explain this in more detail. What is the value of this setting and what does it mean.